tests/conformance/getBlock_test.go

```diff
@@ -0,0 +1,135 @@
+package tests
```

Comment:
Tiny Nit: maybe rename this package "conformance", and the file discovery.go (the test is implicit given it belongs in the tests directory). Same for getBlock.go to "get_block.go".